### PR TITLE
merge #19.34 - Add PHP 7.0 & bulk support

### PR DESCRIPTION
**Pull request by tpellegatta**
### Refers to
Redmine ticket : #[19](https://redmine.bovpg.net/issues/19)

### CODE REVIEW

File: src/Restforce.php [hunk 1]
```diff
@@ -53,28 +82,169 @@ public function __construct(
...
+    /**
+     * Create bulk job
+     *
+     * @param string $object    object to work with
+     * @param string $operation operation
+     *
+     * @return string
+     * @throws Exception
+     */
+    public function createJob(string $object, string $operation)
```
Comment (adefretin) [inline src/Restforce.php]: create.  BulkJob ?    Job => BulkJob

File: src/Restforce.php [hunk 2]
```diff
@@ -88,45 +258,103 @@ public function find(string $sobjectType, string $sobjectId, array $fields = [])
...
+     *
+     * @return OAuthRestClient|null
+     */
+    private function getOAuthRestClient()
     {
         if ($this->oAuthRestClient === null) {
             $this->oAuthRestClient = new OAuthRestClient(
                 new SalesforceRestClient(
-                    new GuzzleRestClient('https://na1.salesforce.com'),
+                    new GuzzleRestClient($this->apiEndpoint),
```
Comment (adefretin) [inline src/Restforce.php:346]: not same url with above

File: src/Restforce.php [hunk 3]
```diff
@@ -53,28 +82,169 @@ public function __construct(
...
+     *
+     * @return string
+     * @throws Exception
+     */
+    public function createJob(string $object, string $operation)
+    {
+        $uri = 'jobs/ingest';
+
+        $job = $this->getOAuthRestClient()->postJson($uri, [
+            'operation' => $operation,
```
Comment (adefretin) [inline src/Restforce.php:120]: could create const for operation values   BULK_JOB_OPERATION_UPDATE / CREATE

File: src/Restforce.php (hunk 2 above)
Comment (tpellegatta) [reply src/Restforce.php:346]: ?

File: src/Restforce.php (hunk 1 above)
Comment (tpellegatta) [reply src/Restforce.php]: It implies modifying these too:

createBulkJob
addBatchToBulkJob
executeBulkJob
checkBulkJob
closeBulkJob

**Pull request merged**